eng/targets/Cpp.Common.targets

```
@@ -12,4 +12,7 @@
   <Target Name="Pack" />
   <Target Name="Restore" />
   <Target Name="ResolveNuGetPackageAssets" />
+  <Target Name="Test" DependsOnTargets="Build" Condition="'$(IsTestProject)' == 'true'" >
```

```suggestion
  <Target Name="Test" Condition="'$(IsTestProject)' == 'true'" >
```
I'd recommend removing this target dependency so `build.cmd -nobuild -test` will work (cref #7137)